app/views/layouts/_header.html.slim

```diff
@@ -8,6 +8,8 @@ header
         li = application_disambiguation_link
       - if current_user.try :current_student?
         li = link_to 'Status Updates', [:students, :status_updates], class: params[:controller] == 'students/status_updates' ? 'active' : ''
+      - if current_user.try :supervisor?
```

Comment:
I know it's already there (e.g. the current_student part), but `try` really isn't needed. If you have a list of items that should only be shown when a user is logged in you can simply wrap all of them in a `- if current_user` block.